src/objects/conf/base.py

```diff
@@ -68,12 +69,7 @@
     # 'django.contrib.sitemaps',
     # External applications.
     "axes",
-    "django_better_admin_arrayfield",
     "drf_spectacular",
-    "sniplates",
-    "hijack",
```

Comment:
I see in the description that it takes over PR with removing hijack. A separate commit for it would be nice. For now a short explanation in the description would be appreciated.

Reply:
Followed https://github.com/maykinmedia/objects-api/pull/343